src/optimize/base_optimizer.js

```diff
@@ -349,6 +349,13 @@ export default class BaseOptimizer {
             test: /\.(html|tmpl)$/,
             loader: 'raw-loader',
           },
+          // NOTE: that rule prevents the bundling of json templates on canvas
+          {
+            test: /\.json$/,
+            include: /[\/\\]plugins[\/\\]canvas[\/\\]canvas_plugin_src[\/\\]templates[\/\\].*\.json$/,
```

Comment:
Can we prefix imports with `!!file-loader!` instead? Hard coding loaders into the base webpack config for specific plugins is something I'd like to avoid if possible.